impl/meson.build

```diff
@@ -148,6 +148,31 @@ if library_type == 'static' and isWindows
 	libSubstrateArgs = ['-DSUBSTRATE_BUILD_STATIC']
 endif
 
+if isWindows
+	rc_cfg = configuration_data()
```

Comment:
`rcConfig` or something similar? The point isn't the exact name suggestion, it's that these need lowerCamelCase'ing to match the style of the rest of the file